includes/class-newspack-popups-inserter.php

```diff
@@ -88,7 +88,7 @@ public static function popups_for_post() {
 		$popups_to_maybe_display = array_filter(
```

Comment:
There's a comment on the line above that should be updated.

Reply:
No longer relevant since this is removed in b8344de.